APIXunitCSharp1/Tests/APITests.cs

```diff
@@ -1,19 +1,61 @@
 using APITestProject.Response;
 using APITestProject.Requests;
+using APITestProject.common;
+using System.Net;
+using System.Xml.Linq;
+using APIXunitCSharp1.Requests;
+using APIXunitCSharp1.Models.Request;
+using APIXunitCSharp1.common;
 
 namespace APITestProject.Tests
 {
     public class APITests
     {
-        [Fact]
-        public void Test1()
+        string baseUrl = Constants.APIBaseURL;
+
+        IUsersAPIMethods usersApi { set; get; }
+        public APITests()
+        {
+            usersApi = new UsersAPIMethods(baseUrl);
+        }
+        [Theory]
+        [InlineData("/api/users")]
+        public void UsersList(string apiUrl)
         {
-            var list = new UsersAPIMethods();
-            var response = list.GetUsers();
+            //var apiMethods = new UsersAPIMethods<AllUsersResponse>(baseUrl);
+            var response = usersApi.GetUsers(apiUrl);
+            if(response.StatusCode == HttpStatusCode.OK)
+            {
+                var dataResponse = response.Data;
+                if(dataResponse != null) {
+                    Assert.Equal(6, dataResponse.per_page);
```

Comment:
Having static expected data may be a breaking thing. please consider other assertions like ensuring the keys present and schema of the response in cases where we are not sure of the response data.